src/BikramSambat.ts

```diff
@@ -0,0 +1,22 @@
+import { NepaliDaysData } from "data/nepali-days";
+import { NepaliMonthsData } from "data/nepali-months";
+import { NewYearMappingData } from "data/new-year-mapping";
+import { DaysInMonthsMappingData } from "data/days-in-month-mapping";
```

Comment:
suggestion: what do you say if we export all data objects from single `index.ts` file in data folder, that way we will have single point of data imports and no worry of messing up of import paths